src/pages/Home/components/HomeHeader/HomeHeader.vue

```diff
@@ -10,9 +10,7 @@
         <p>Código: {{ memberCode }}</p>
         <p
           :class="`Header__membership ${
-            isMembershipActive
-              ? 'Header__membership--active'
-              : 'Header__membership--inactive'
+            isMembershipActive || 'Header__membership--inactive'
```

Comment:
I don't think this is good it will be "false" in some cases, and that is not a valid CSS class

@yreyes2903 please check

Reply:
restore after deeming it necessary to highlight the state with colors